natives.json

```diff
@@ -28277,18 +28277,18 @@
 		},
 		"0x7ABE7095FB3D2581": {
 			"name": "_PROMPT_SET_ROTATE_MODE",
-			"comment": "",
+			"comment": "p1 is how much to rotate. p2 dictates if the rotation should be CCW",
 			"params": [
 				{
-					"type": "Any",
-					"name": "p0"
+					"type": "Prompt",
+					"name": "prompt"
 				},
 				{
-					"type": "Any",
+					"type": "Integer",
 					"name": "p1"
 				},
 				{
-					"type": "Any",
+					"type": "Boolean",
```

Comment:
And this should be an ``int`` (name: ``direction``).

Reply:
I can promise it's not an int because when used ingame the int did nothing.

Reply:
I'll rephrase that setting it to an int anywhere from 1 to 10 resulted in the prompt asking for a CCW rotation. When it's a bool it then asks for CCW or CW. Setting this param to 0 or anything higher than 10 causes the native to fail. So please tell me what it should be again. :)

Reply:
And again, it is definitely an int, see my comment above.